src/main/kotlin/com/sourcegraph/cody/agent/protocol/ContextItem.kt

```diff
@@ -44,7 +53,8 @@ data class ContextItemFile(
     val source: ContextFileSource? =
         null, // Oneof: embeddings, user, keyword, editor, filename, search, unified, selection,
     // terminal
-    val content: String? = null
+    val content: String? = null,
+    val isTooLarge: Boolean? = null
```

Comment:
NOTE: this field does not exist yet but I see it is there in the later cody versions